CIP-0008/README.md

```diff
@@ -388,26 +289,151 @@ We recommend usage of unprotected headers vs protected headers when possible. Th
+
+#### Other blockchain standards
+
+Other blockchains have existing specifications for message signing, but they mostly revolve around scripts trying to validate messages. We don't leverage any of their work in particular but it may be of interest.
+
+- [BIP-137](https://github.com/bitcoin/bips/blob/master/bip-0137.mediawiki) - simply scheme for message signing that works with P2PKH, P2PSH and bech32
+- [BIP-322](https://github.com/kallewoof/bips/blob/master/bip-0322.mediawiki) - reuses Bitcoin script to process a generic signed message format
+
+- [EIP-191](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-191.md) - encode data for Ethereum contracts
+- [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md) - encode structs for Ethereum contracts
+
+### Existing Code
+
+#### Cryptography
+
+Cardano already allows message signing within the WASM bindings. Notably,
+
+1) [sign](https://github.com/Emurgo/cardano-serialization-lib/blob/4792b1b121e728a51686d5fcbffd33489d65c903/rust/src/crypto.rs#L279)
+2) [verify](https://github.com/Emurgo/cardano-serialization-lib/blob/4792b1b121e728a51686d5fcbffd33489d65c903/rust/src/crypto.rs#L322)
+
+You can see an example of these two functions [here](https://repl.it/repls/FlusteredSimpleFunction)
+
+Even if you use cryptographically secure `sign` and `verify` functions, you still have the following problems:
+
+1) No human-recognizable prefix
+0) No error detection
+0) User could accidentally sign a transaction or a block thinking it's harmless data
+
+We also have a risk of a few different kinds of replay attacks
+
+4) A dapp asks person A to sign "BOB" and then another dapp asks user B to sign "BOB". B can just use the signature from A
+0) A dapp asks person A to sign "BOB" on a testnet chain. Person B then sends this signed message to the same dapp running on mainnet (same argument applies to sidechains)
+
+### Reference implementations
 
 - [COSE-JS](https://github.com/erdtman/cose-js)
 - [Rust message signing](https://github.com/Emurgo/message-signing)
 
 ### Unresolved
 
-This specification leaves the following unresolved
+This specification provides no means of Revocation.
 
-- Revocation
+## Path to Active
 
-### Examples
+### Acceptance Criteria
 
-TODO
+- [x] There are wallets supporting creation of signed messages as per this protocol (enumerated 2023-12-19 as per [CardanoBallot list of wallets](https://voting.summit.cardano.org/user-guide) supporting CIP-8 signed messages):
+  - [x] Flint
+  - [x] Eternl
+  - [x] Nami
+  - [x] Typhon
+  - [x] Yoroi
+  - [x] Nufi
+  - [x] Gerowallet
+  - [x] Lace
+- [x] There exist one or more implementations in commonly used development libraries:
+  - [x] [Mesh](https://meshjs.dev/about/cips)
+  - [x] `@emurgo/cardano-message-signing-asmjs`
+- [x] There exist one or more implementations in web sites and other tools:
+  - [x] SundaeSwap Governance voting
```

Comment:
Would be nice to also add CLI Tools here like:
- [x] CLI Tools supporting creation and verification of signes messages
  - [x] [Cardano-Signer](https://github.com/gitmachtl/cardano-signer)

Reply:
thanks @gitmachtl ... added in https://github.com/cardano-foundation/CIPs/pull/648/commits/787bc273a2044bbc633247c20e7674c1800471a1